modules/sharedaddy/sharing-sources.php

```diff
@@ -1602,9 +1602,9 @@ function jetpack_sharing_pocket_init() {
 
 }
 
-class Share_Telegram extends Sharing_Source {
-	public $shortname = 'telegram';
-	public $genericon = '\f224';
+class Share_TelegramMessenger extends Sharing_Source {
+	public $shortname = 'telegramme';
```

Comment:
Actually, this was intentional, matching the existing shortname / service name usage. Comparison below. Let me know if I should change it though!

Shortnames: 
```php
#$ ag "shortname = '" modules/sharedaddy/sharing-sources.php
359:	public $shortname = 'email';
538:	public $shortname = 'twitter';
734:	public $shortname = 'reddit';
769:	public $shortname = 'linkedin';
845:	public $shortname = 'facebook';
978:	public $shortname = 'print';
999:	public $shortname = 'pressthis';
1061:	public $shortname = 'googleplus1';
1349:	public $shortname = 'tumblr';
1395:	public $shortname = 'pinterest';
1544:	public $shortname = 'pocket';
1606:	public $shortname = 'telegramme';
```

[Services:](https://github.com/Automattic/jetpack/blob/b008265fce5d81cee8e731a7c40319a8c84e6445/modules/sharedaddy/sharing-service.php#L47-L58)
```php
      'email'         => 'Share_Email',
      'print'         => 'Share_Print',
      'facebook'      => 'Share_Facebook',
      'linkedin'      => 'Share_LinkedIn',
      'reddit'        => 'Share_Reddit',
      'twitter'       => 'Share_Twitter',
      'press-this'    => 'Share_PressThis',
      'google-plus-1' => 'Share_GooglePlus1',
      'tumblr'        => 'Share_Tumblr',
      'pinterest'     => 'Share_Pinterest',
      'pocket'        => 'Share_Pocket',
      'telegram-me'   => 'Share_TelegramMessenger',
```